etc/profile-m-z/tesseract.profile

```diff
@@ -0,0 +1,63 @@
+# Firejail profile for tesseract
+# Description: An OCR program
+# This file is overwritten after every install/update
+# Persistent local customizations
+include tesseract.local
+# Persistent global definitions
+include globals.local
+
+blacklist ${RUNUSER}
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+whitelist ${DOCUMENTS}
+whitelist ${DOWNLOADS}
+whitelist ${PICTURES}
+include whitelist-common.inc
+include whitelist-run-common.inc
+include whitelist-runuser-common.inc
+whitelist /usr/share/tessdata
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+hostname tesseract
+ipc-namespace
+machine-id
+net none
+no3d
+nodvd
+nogroups
+noinput
+nonewprivs
+noprinters
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix
```

Comment:
Is unix used or can we use `seccomp socket`?

Reply:
I'll be testing this shortly. Thanks for the review and your top-notch remarks/suggestions.

Reply:
`seccomp socket` works just fine. Added!